### ISSUE TITLE
GTA VCS missing jaw characters IOS

### ISSUE THREAD

**Issue by BlackGor** [hrydgard/ppsspp#14566]
Hi ! I have try the IOS version of PPSSPP and i play GTA VCS ,  but there are a strange glitch during cutscenes , characters are missing textures (jaws are missing and we see only teeths) , i'm on IOS 14.6.

**Comment by Gamemulatorer** [comment]
Duplicate https://github.com/hrydgard/ppsspp/issues/14514

**Comment by hrydgard** [comment]
Closing as duplicate. Known issue, will be investigated soon.
